routes/v1/gardens/gardens_router.go

```diff
@@ -0,0 +1,33 @@
+package gardens_router
+
+import (
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gorilla/mux"
+	gardens_controller "github.com/ice-cream-backend/controller/v1/gardens"
+)
+
+func GardensIdGet(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
```

Comment:
@yukari-n-erb You can add this after my push but you'll need to add CORS check so that the frontend can access the endpoints. I made a func in the utils so please add it in after my merge is done!